packages/replay/metadata/test.ts

```diff
@@ -1,3 +1,5 @@
+import { Struct } from "superstruct";
```

Comment:
:hand: can probably use `import type { Struct }` here. Not critical, but nice to delineate what's a type vs what's a value